### ISSUE TITLE
non-javascript <script> contents parsed as html

### ISSUE THREAD

**Issue by stevenvachon** [inikulin/parse5#84]
```js
parse5.parseFragment('<script type="text/foo"><tag attr/></script>');
```
The contents should be a string, correct? v1.5 did this. v2.0 is parsing <tag> as HTML.

**Comment by inikulin** [comment]
Right, thank you for finding this. Ironically, it's not an issue with the parsing algorithm, but another case of  the #82

**Comment by inikulin** [comment]
Published as v2.0.2

**Comment by stevenvachon** [comment]
Awesome! Thanks!!
